src/components/SearchBy/SearchBy.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import styles from './SearchBy.scss';
+
+const cx = classNames.bind(styles);
+
+class SearchBy extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchBy: 'title',
+    };
+  }
+
+  handelSearch(e, val) {
+    e.preventDefault();
+    switch (val) {
```

Comment:
You don't need switch. Just use default value for val param and call this.setState({ searchBy: val }) only once